modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/jobs/jobs.js

```diff
@@ -15,31 +15,29 @@ var registry = require("platform/v4/registry");
 exports.Job = function Job(constructJob) {
   if (!constructJob.uri) throw "URI not specified";
 
-  this.path = constructJob.uri;
-
-  com.sap.xsk.xsjob.ds.facade.XSKJobFacade.newJob(registry.getText(this.path), this.path);
+  this.names = com.sap.xsk.xsjob.ds.facade.XSKJobFacade.newJob(registry.getText(constructJob.uri));
```

Comment:
What `this.names` stands for? Shouldn't it be `this.schedules`?

Reply:
I've called it names since each schedule is a separate job and that job is referenced by its name. I see why it may seem confusing in the context of this API, though, I'll change it.